content/kotlin/concepts/strings/terms/dropLast/dropLast.md

```diff
@@ -0,0 +1,46 @@
+---
+Title: 'dropLast()'
+Description: 'Returns a string with n characters removed.'
+Subject:
+  - 'Computer Science'
+Tags:
+  - 'Methods'
+  - 'Data Types'
+  - 'Characters'
+CatalogContent:
+  - 'learn-kotlin'
+  - 'paths/computer-science'
+---
+
+The Kotlin method, **`.dropLast()`** removes the last character of a string or the last `n` number of characters from a string. This method does not modify the original list, it creates and returns a new list.
+
+## Syntax
+
+```pseudo
+string.dropLast(2)
```

Comment:
```suggestion
fun String.dropLast(n: Int): String
```
As it is a syntax, there should not be a specific value in the place of a parameter. This looks more of a proper syntax, please have a look at it.